server/images.py

```diff
@@ -0,0 +1,59 @@
+import datetime
+import os
+
+import pytz
+
+# Format of GreenPiThumb image files.
+_FILENAME_FORMAT = '%Y-%m-%dT%H%MZ.jpg'
+
+
+def _files_in_directory(directory):
+    paths = os.listdir(directory)
+    return [f for f in paths if os.path.isfile(os.path.join(directory, f))]
+
+
+def _timestamp_from_filename(filename):
+    return datetime.datetime.strptime(filename, _FILENAME_FORMAT).replace(
+        tzinfo=pytz.utc)
+
+
+def _filename_to_index_entry(filename):
+    return {
+        'timestamp': _timestamp_from_filename(filename),
+        'filename': filename,
+    }
+
+
+class Indexer(object):
+    """Creates an index of GreenPiThumb image files."""
+
+    def __init__(self, image_path):
+        """Creates a new Indexer instance
```

Comment:
(optional) ```image_path``` sounds to me like it could be the path of a specific image. Maybe ```images_path``` or something else is a little clearer.